games/games.controller.js

```diff
@@ -14,21 +15,35 @@ router.get("/", async (req, res) => {
 });
 
 router.post("/", async (req, res) => {
-  const { name, genreId } = req.body;
-  const genre = await getGenre(genreId);
+  const { name, genres, description, ageRating } = req.body;
 
-  const game = await addGame(name, genre);
+  const genreObjects = [];
+
+  genres.map(async g => {
+    let genre = await getGenre(g);
+    genreObjects.push(genre);
+  });
+
+  const game = await addGame(name, genreObjects, description, ageRating);
 
   res.send(game);
 });
 
 router.put("/:id", async (req, res) => {
-  const { name, genreId } = req.body;
-  const genre = await getGenre(genreId);
+  const { name, genres, description, ageRating } = req.body;
+
+  const genreObjects = [];
+
+  genres.map(async g => {
+    let genre = await getGenre(g);
+    genreObjects.push(genre);
+  });
 
   const game = await updateGame(req.params.id, {
     name,
-    genre
+    genreObjects,
```

Comment:
This should be genres: genreObjects instead. We are creating an object that is genreObjects: genreObjects which of course wouldn't work. This will be the same for the post method too.